### DISCUSSION TITLE
DateComparison

### DISCUSSION THREAD

**Discussion by mb6852** [moj-analytical-services/splink#2399] [General]
What is the preferred method to perform comparison on a date column?

**Comment by RobinL** [comment]
To use date comparisons in Splink, first import the comparison library:

```python
import splink.comparison_library as cl
```

Two useful comparisons for dates are:

1. `cl.DateOfBirthComparison()`
2. `cl.AbsoluteDateDifferenceAtThresholds()`

These can be used in your Splink settings to compare date fields with various levels of precision and tolerance.

**Comment by mb6852** [comment]
Thank you for your reply. AbsoluteDateDifferenceAtThresholds() is not listed here https://moj-analytical-services.github.io/splink/api_docs/comparison_library.html but it is included.

**Reply by RobinL** [reply]
Thanks.  It's because they aliases.  Fixed here:
https://github.com/moj-analytical-services/splink/pull/2402/files
